habitat-lab/habitat/config/read_write.py

```diff
@@ -3,23 +3,21 @@
 # LICENSE file in the root directory of this source tree.
 
 from contextlib import contextmanager
-from typing import TYPE_CHECKING, Generator
+from typing import Any, Generator
 
-from omegaconf import OmegaConf
+from omegaconf import Container, OmegaConf
 from omegaconf.base import Node
 
-if TYPE_CHECKING:
-    from omegaconf import Container
-
 
 @contextmanager
-def read_write(config: "Container") -> Generator[Node, None, None]:
+def read_write(config: Any) -> Generator[Node, None, None]:
```

Comment:
We should probably avoid using this altogether.

Changing configs mid-execution contributes to the complexity explosion we're experiencing, and drives us away from a data-driven paradigm.